persistence/mongodb-record/src/main/scala/net/liftweb/mongodb/record/field/MongoPk.scala

```diff
@@ -73,8 +73,9 @@ trait StringPk[OwnerType <: MongoRecord[OwnerType]]
   self: OwnerType =>
 
   def defaultIdValue = StringHelpers.randomString(32)
```

Comment:
Just a tiny observation- should the defaultIdValue use the maxIdLength constant instead of the number?

Reply:
Probably a good idea.